packages/babel-parser/src/parser/statement.js

```diff
@@ -1373,6 +1373,7 @@ export default class StatementParser extends ExpressionParser {
       prop.computed = false;
       prop.key = key;
       prop.static = false;
+      prop.accessor = false;
```

Comment:
This should be more like
```js
if (this.hasPlugins("decorators") && this.getPluginOption("decorators", "version") === "X")
```
where "X" is a name we give to the current version (it could even just be the date of the last meeting where the current proposal was presented).